drivers/vmwarevsphere/cloudinit.go

```diff
@@ -166,27 +167,41 @@ func (d *Driver) createCloudInitIso() error {
 			return fmt.Errorf("error: %s found when verifying that %s file was present for machine %s", err, filename, d.MachineName)
 		}
 	}
+	machineDir := filepath.Join(d.StorePath, "machines", d.MachineName)
```

Comment:
Is it guaranteed that this directory exists at this point?

Reply:
Yes, it gets created above https://github.com/rancher/machine/blob/3b147adc9e9a6aefb074f35c76af3897a271fc7d/drivers/vmwarevsphere/cloudinit.go#L148-L151